src/main.rs

```diff
@@ -32,17 +40,33 @@ struct ComponentCode {
     code: String,
 }
 
-fn handle_request(account_id: &str, path: PathBuf) -> FileList {
+fn handle_request(account_id: &str, path: PathBuf, network: Network) -> FileList {
     let mut components = HashMap::new();
-    get_file_list(&path, account_id, &mut components, String::from(""));
+    get_file_list(&path, account_id, &mut components, String::from(""), network);
     FileList { components }
 }
 
+fn replace_placeholders(code: &str, account_id: &str, network: Network) -> String {
+    let mut replacements = HashMap::new();
+    replacements.insert("${REPL_ACCOUNT}", account_id);
+    replacements.insert("${REPL_NEAR_URL}", if network == Network::Testnet {"test.near.org"} else {"near.org"});
+    replacements.insert("${REPL_NEAR_SOCIAL_ACCOUNT}", if network == Network::Testnet {"v1.social08.testnet"} else {"social.near"});
```

Comment:
this one is fine to handle in this way since everyone uses the same contract regardless of gateway. Please rename to `REPL_SOCIALDB_CONTRACT` since SocialDB is the smart contract whereas Near Social is the gateway

Reply:
Great remark. I fixed it in this PR as well: https://github.com/near/near-discovery-components/pull/107